config/config.go

```diff
@@ -398,6 +391,21 @@ func getDefaultStringArray(envvar string, defaultValue string) (retVal []string)
 	return
 }
 
+func getDefaultStringMap(envvar string, defaultValue string) (retVal map[string]string) {
+	csv := os.Getenv(envvar)
+	if csv == "" {
+		csv = defaultValue
+	}
+	retVal = map[string]string{}
+	for _, token := range strings.Split(csv, ",") {
+		mapEntry := strings.Split(token, "=")
```

Comment:
you should force this to split with a max of 2 using `SplitN` (I don't think namespaces can have = chars, but still we should force this to only split on the first = it sees anyway).
```suggestion
		mapEntry := strings.SplitN(token, "=", 2)
```

Reply:
Good tip, changed to SplitN.